cypress/integration/navigation/index.spec.js

```diff
@@ -0,0 +1,134 @@
+import { raagIndices } from '../../../src/js/pages/GranthIndex/constants/raag-indices';
+
+const getRandomNumber = (max) => Math.floor(Math.random() * Math.floor(max));
+
+Cypress.Commands.add('checkGranthIndices', function checkGranthIndices({ granthIndex, indices, source }) {
```

Comment:
should this go in `cypress/support/commands.js` ?

Reply:
I've seen it where the index file under support contains common commands and then a file for each specific suite has custom commands for only that suite. It would make reading the test file easier

Reply:
@tsingh777 thanks a lot veerji for the suggestion. Yeah since this was custom command so I just kept it enclosed but 100% agree with you that way it makes code easier to read.